### PR TITLE
tweak abstract and intro

### PR DESCRIPTION
**Pull request by danwing**

### CODE REVIEW

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 1]
```diff
@@ -76,10 +76,11 @@ at the network (e.g., packet discard preference), the sender (e.g.,
 adaptive transmission or session migration), or through cooperation of both the host
 and the network.
 
-This document outlines a set of use-cases that highlight the need for a mechanism
-to share metadata about flows between a host and its network in order to enable different traffic treatment.
-Such a mechanism is typically implemented using a signaling protocol between the host and
-a set of trusted netwrok elements.
+This document lists uses-cases demonstrating the need for a mechanism
+to share metadata about flows between a receiving host and its network
+to enable different traffic treatment for packets sent to the
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:81]:
```suggestion
to enable differentiated traffic treatment for packets sent to the
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 2]
```diff
@@ -76,10 +76,11 @@ at the network (e.g., packet discard preference), the sender (e.g.,
 adaptive transmission or session migration), or through cooperation of both the host
 and the network.
 
-This document outlines a set of use-cases that highlight the need for a mechanism
-to share metadata about flows between a host and its network in order to enable different traffic treatment.
-Such a mechanism is typically implemented using a signaling protocol between the host and
-a set of trusted netwrok elements.
+This document lists uses-cases demonstrating the need for a mechanism
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:79]:
```suggestion
This document lists uses cases demonstrating the need for a mechanism
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 3]
```diff
@@ -96,7 +97,10 @@ The simple network diagram below shows where such bandwidth and
 performance constraints usually exist with a "B" (for Bottleneck).
 Other network bottlenecks may be experienced in other segments not shown
 in the figure, such as interconnection links or the infrastructure that hosts the service (e.g.,
-flash crowds). A bottleneck may be limited in time, present or not regular patters, etc.
+flash crowds).  When a bottleneck exists temporarily, the network has no choice but
+to discard or delay packets -- which can harm certain flows.  In this
+paper, this is termed 'reactive policy'.
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:102]:
```suggestion
document, this is termed 'reactive policy'.
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 4]
```diff
@@ -96,7 +97,10 @@ The simple network diagram below shows where such bandwidth and
 performance constraints usually exist with a "B" (for Bottleneck).
 Other network bottlenecks may be experienced in other segments not shown
 in the figure, such as interconnection links or the infrastructure that hosts the service (e.g.,
-flash crowds). A bottleneck may be limited in time, present or not regular patters, etc.
+flash crowds).  When a bottleneck exists temporarily, the network has no choice but
+to discard or delay packets -- which can harm certain flows.  In this
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:101]:
```suggestion
to discard or delay packets -- which can harm certain flows and thus lead to suboptimal perceived experience.  In this
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 5]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
-to encrypt or integrity protect the metadata with a key; such a key
-could be established using a signaling protocol, see {{key}}.
-
-There is some consensus that applications can benefit by collaborative signaling
+* for network elements:
+  * Which metadata to honor
+  * Tradeoff between the extra cost (including processing) versus benefits
+  * Operational impacts
+
+Configured cooperation between an ISP and content providers can allows
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:141]:
```suggestion
Configured cooperation between an ISP and content providers (via contractual agreements, typically) can allow
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 6]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
+metadata signals augmenting packets to be honored by the ISP.  This
+cooperation has historically involved examination of server IP
+address, TLS SNI, AS number, or heuristics to identify flows.
+However, this cooperation favors large Internet service providers and
+favors large content providers.  Smaller ISPs, small content
+providers, and new content providers are harmed.
+
+A more egalitarian approach provides the same benefit ot parties --
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:149]:
```suggestion
A more egalitarian approach provides the same benefit to parties --
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 7]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
+large and small -- and also provide richer signaling to further
+improve user experience and metadata interoperability. This allows all
+parties to become part of the "Internet fast lane".
+
+Rather than relying on configured cooperation between ISPs and
+content providers, this paper shows use-cases where the client
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:155]:
```suggestion
content providers, this document shows use cases where the client
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 8]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
+tells its ISP the importance of packets it is receiving.  This
+provides several benefits described in later sections.
+
+There is already some consensus that applications can benefit by collaborative signaling
 the network ({{?IAB=RFC9419}}, {{ATIS}}).  This document provides
-use-cases to further detail the need of such signaling.
+use-cases to further detail the need of such signaling and highlights
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:161]:
```suggestion
use cases to further detail the need of such signaling and highlights
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 9]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
+* for network elements:
+  * Which metadata to honor
+  * Tradeoff between the extra cost (including processing) versus benefits
+  * Operational impacts
+
+Configured cooperation between an ISP and content providers can allows
+metadata signals augmenting packets to be honored by the ISP.  This
+cooperation has historically involved examination of server IP
+address, TLS SNI, AS number, or heuristics to identify flows.
+However, this cooperation favors large Internet service providers and
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:145]:
```suggestion
However, this cooperation might have scalability issues (e.g., ISPs to establish and maintain agreements with a large number of content providers), operational implications as the network configuration will rely on service-specific (e.g., risk of frozen or stale configuration), mismatch between the expected service level from users and the delivered one, etc. 
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 10]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
+favors large content providers.  Smaller ISPs, small content
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:146]:
```suggestion
Moreover, smaller ISPs, small content
```

File: draft-rwbr-tsvwg-signaling-use-cases.md [hunk 11]
```diff
@@ -122,53 +126,41 @@ that terminate a bandwidth constraining link (or located few hops next to that e
...
+providers, and new content providers are harmed.
+
+A more egalitarian approach provides the same benefit ot parties --
+large and small -- and also provide richer signaling to further
+improve user experience and metadata interoperability. This allows all
+parties to become part of the "Internet fast lane".
```
Comment (boucadair) [inline draft-rwbr-tsvwg-signaling-use-cases.md:152]:
```suggestion
parties to become part of the "Internet fast lane". Also, this approach improves the Internet for end users per {{?RFC8890}}.
```

**Review by boucadair — CHANGES_REQUESTED** [review]

**Review by danwing — COMMENTED** [review]
Thanks, Med!

**Comment by boucadair** [comment]
Is it normal that none of the proposed changes are implemented?

**Comment by danwing** [comment]
I’ll take a look. I did a bulk commit for first time maybe I did that wrong. -dOn Mar 13, 2024, at 9:11 AM, Med ***@***.***> wrote:﻿
Is it normal that none of the proposed changes are implemented?

**Pull request merged**